fire/cli/niv/__init__.py

```diff
@@ -286,16 +286,22 @@ def find_faneblad(
     projektnavn: str, faneblad: str, arkdef: Dict, ignore_failure: bool = False
 ) -> pd.DataFrame:
     try:
-        return (
-            pd.read_excel(
-                f"{projektnavn}.xlsx",
-                sheet_name=faneblad,
-                usecols=anvendte(arkdef),
+        raw = pd.read_excel(
+            f"{projektnavn}.xlsx",
+            sheet_name=faneblad,
+            usecols=anvendte(arkdef),
+        ).dropna(how="all")
+
+        if set(raw.columns) ^ set(arkdef):
+            fire.cli.print(
+                f"Kolonnenavne i fanebladet '{faneblad}' matcher ikke arkdefinitionens\n\n    {list(arkdef.keys())}\n"
             )
-            .dropna(how="all")
-            .astype(arkdef)
-            .replace("nan", "")
-        )
+            fire.cli.print(f"Undersøg eventuelt, om der er dubletter i kolonnenavnene.")
+            # fire.cli.print("Pandas overskriver kolonnenavne, der findes flere gange, så vi kan kun checke, om der mangler noget i forhold til arkdefinitionen.")
```

Comment:
Skulle det her i virkeligheden have været en rigtig kommentar og ikke et print-statement?

Reply:
Jeg ville gerne fortælle brugeren, hvorfor programmet ikke kan se, når problemet skyldes to eller flere kolonner med samme navn, og brugeren derfor skal bruge sine egne øjne. På den anden side er det en mere teknisk kommentar, der måske forvirrer mere, end den gavner, da dykker ned i implementeringen. Jeg savnede lidt, at man kan slå mere forklaring til med et ekstra flag til kommandoerne.

**Jeg gør beskrivelsen mindre implementationsspecifik og giver brugeren info om de forventedekolonner.**